c7n/resources/efs.py

```diff
@@ -137,3 +139,55 @@ def unmount_filesystems(self, resources):
             for t in client.describe_mount_targets(
                     FileSystemId=r['FileSystemId'])['MountTargets']:
                 client.delete_mount_target(MountTargetId=t['MountTargetId'])
+
+
+@ElasticFileSystem.action_registry.register('configure-lifecycle-policy')
+class ConfigureLifecycle(BaseAction):
+    """Enable/disable lifecycle policy for efs.
+
+    :example:
+
+      .. code-block:: yaml
+
+            policies:
+              - name: efs-apply-lifecycle
+                resource: efs
+                actions:
+                  - type: configure-lifecycle-policy
+                    state: enable
+                    rule:
+                      - 'TransitionToIA': 'AFTER_7_DAYS'
+
+    """
+    schema = type_schema(
+        'configure-lifecycle-policy',
+        state={'enum': ['enable', 'disable']},
+        rule={'type': 'array'},
```

Comment:
rule should be plural if its an array, ie `rules`

Reply:
Sounds good! Thanks!